.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think we can remove this.

Reply:
I normally do `git add -p file-i-want-to-add` to avoid adding everything. Instead of `git add . `